ext/bg/js/dictionary.js

```diff
@@ -123,7 +123,7 @@ function dictTermsCompressTags(definitions) {
 }
 
 function dictTermsGroup(definitions, dictionaries) {
-    const groups = {};
+    const groups = new Map();
     for (const definition of definitions) {
         const key = [definition.source, definition.expression];
         key.push(...definition.reasons);
```

Comment:
Also unrelated to this PR, but should this be sorted?

Reply:
I'm not sure; there's a `dictTermsSort` call at the end, but I don't know if it's required or what its exact purpose is (I haven't looked into it either though).